SU-HSDO/suhumsci
# SHS-5643 rethink the purpose of alt text listing

# READY FOR REVIEW

## Summary
https://fourkitchens.clickup.com/t/36718269/SHS-5643
A few tweaks to Views config to fix misc bugs.

## Need Review By (Date)
end of sprint

## Urgency
normal

## Steps to Test
Visit `/admin/content/alt`
1. are the exposed filters clear?  
2. Does click-sorting work on all columns (note that some fields might have a leading tab that could confuse you)?  
3. Try to visit the page while logged out, do you get a 403?

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
